casts/countryPicker/src/UICountryPicker/CountryPicker.tsx

```diff
@@ -121,6 +129,13 @@ export function CountryPicker({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    const hideKeyboard = React.useCallback(() => {
+        // react-native-android-keyboard-adjust bug:
```

Comment:
Not sure that it's a bug of the lib, I suspect it's sth with `keyboardDismissMode`

Reply:
@savelichalex I wrote incorrectly. In fact, it closes, but with a long delay, when I use keyboardDismissMode="on-drag"

Reply:
Have you tried https://reactnative.dev/docs/scrollview#keyboardshouldpersisttaps?

Reply:
yes